### ISSUE TITLE
Throwing an `AssertionError.` does not return when using nREPL

### ISSUE THREAD

**Issue by MrEbbinghaus** [babashka/babashka.nrepl#48]
**version** v0.6.7
**platform** macOS
**problem**
Throwing an `AssertionError.` does not return when connected to bb nREPL.
This breaks [`assert`](https://clojuredocs.org/clojure.core/assert).

**repro**
```sh-session 
❯ bb nrepl-server
Started nREPL server at 127.0.0.1:1667
For more info visit: https://book.babashka.org/#_nrepl
```

```sh-session
❯ bb
Babashka v0.6.7 REPL.
Use :repl/quit or :repl/exit to quit the REPL.
Clojure rocks, Bash reaches.

user=> (assert false)
Assert failed: false
user=> (throw (AssertionError. "fail"))
java.lang.AssertionError: fail [at <repl>:2:1]
user=> ^C

❯ lein repl :connect 1667
Connecting to nREPL at 127.0.0.1:1667
user=> (throw (AssertionError. "Fail")

...
forever and ever...

❯ lein repl :connect 1667
Connecting to nREPL at 127.0.0.1:1667
user=> (assert false)

...
forever and ever...
```

**expected behavior**

```sh-session
❯ lein repl :connect 1667
Connecting to nREPL at 127.0.0.1:1667
user=> (assert false)
Assert failed: false
user=>

**Comment by borkdude** [comment]
We should probably catch `Throwable` here instead of `Exception`:

https://github.com/babashka/babashka.nrepl/blob/fa3a55ae58f7bcbcfcfaf10fab98b412220f733f/src/babashka/nrepl/impl/server.clj#L116

**Comment by borkdude** [comment]
@MrEbbinghaus Released in babashka 0.6.8 (along with clojure.core/test).

**Comment by MrEbbinghaus** [comment]
👍🏻   Will come in handy when the Advent of Code tasks get tougher. 😉
